proto/v1/page.proto

```diff
@@ -26,3 +27,19 @@ message ProteinPageResponse {
   string entity = 1;
   datacommons.GraphNodes data = 2;
 }
+
+message PlacePageRequest {
+  string entity = 1;
+  // A list of additional stat vars need to be fetched in addition to the
+  // data in cache. This is to be used in local development, where new chart is
+  // added to chart config but data is not added to cache (delay from cache
+  // build).
+  repeated string new_stat_vars = 2;
+  // Seed value for random selection. Used by test to get deterministic result.
+  int64 seed = 3;
+}
+
+message PlacePageResponse {
+  string entity = 1;
+  datacommons.GetPlacePageDataResponse data = 2;
```

Comment:
Since we only have singular version of this API and it's internal, can just use GetPlacePageDataResponse here (so no need to have the wrapper with entity)?

Reply:
done